helm/qontract-reconcile/templates/template.yaml

```diff
@@ -127,8 +127,102 @@ objects:
+              qr:
+                type: file
+                include:
+                  - /fluentd/log/integration.log
+                read_from: beginning
+                data_dir: /fluentd/log
+
+            transforms:
+              ### De-spaminate
+              despam_events:
+                type: filter
+                inputs:
+                  - qr
+                condition:
+                  type: datadog_search
+                  source: -"using gql endpoint" AND -"Certificate did not match expected hostname"
+
+              ### Craft a JSON document that has escaped double quotes and changes the event from the 
+              ### Vector-default `message` to `text`, which is require by Slack and Google Chat.
+              rename_message_webhooks:
+                type: remap
+                inputs:
+                  - despam_events
+                source: |-
+                  .message = join!(
+                  [
+                    "{\"text\":\"",
+                    format_timestamp!(
+                      now(),
+                      "%Y-%m-%d %H:%M:%S %z"
+                    ),
+                    " ",
+                    " [${POD_NAME}] ",
+                    replace!(
+                      del(.message),
+                      "\"",
+                      "\\\""),
+                    "\"}"
+                  ])
+
+            sinks:
+              ################
+              ### Cloudwatch log sink
+              ################
+              cloudwatch_logs:
+                type: aws_cloudwatch_logs
+                inputs:
+                  - rename_message_webhooks
+                group_name: "${LOG_GROUP_NAME}"
+                encoding:
+                  codec: text
+                stream_name: "{{ $integration.name }}"
+
+            {{- if $logs.slack }}
+              ################
+              ### Slack sink
+              ################
+              slack:
+                type: http
+                inputs:
+                  - rename_message_webhooks
+                compression: none
+                encoding:
+                  codec: text
+                  only_fields:
+                    - message
+                method: post
+                uri: ${SLACK_WEBHOOK_URL}
+            {{- end }}
+
+            {{- if $logs.googleChat }}
+              ################
+              ### Google Chat sink
+              ################
+              googlechat:
+                type: http
+                inputs:
+                  - rename_message_webhooks
+                compression: none
+                encoding:
+                  codec: text
+                  only_fields:
+                    - message
+                method: post
+                uri: https://${GOOGLE_CHAT_WEBHOOK_URL}&threadKey=${POD_NAME}
+            {{- end}}
+            EOF
+            ### APPSRE-8096: Vector Section Ends
+            echo "Creating /fluentd/etc/fluent.conf"
```

Comment:
Is this line important? It wasn't present before?

Reply:
They're here only for the case we find that we must excise Vector during or immediately after the evaluation period. It makes it very clear where the section begins and ends. It probably made more sense before needing to adjust 28 individual fixture files.